src/components/_tooltip.scss

```diff
@@ -0,0 +1,68 @@
+@use "../design-tokens/color";
+@use "../design-tokens/typography";
+@use "../design-tokens/shadow";
+
+$tooltip-arrow-width: 5px;
+$tooltip-color: color.$color-grayDarker;
+
+@mixin Manifold__Tooltip {
+  position: relative;
```

Comment:
This might need to be `absolute`. It's hard to say how exactly this should be used. perhaps we need a `Manifold___Tooltip__Container` that wraps the contents that should trigger the tooltip to appear?

Reply:
Yeah, was chatting with @dangodev about that. I'm not sure what the solution is yet as there are a lot of edge cases to consider with tooltips. This PR could just be the visual aspect if need be.

Reply:
I actually like that idea. Should it be `Container`? Maybe `BoundingBox`? (I like `Container` just wanted to bikeshed on the name a lil).

I think you’d still need a way to manually position these puppies, but that’d call for `position: absolute` either way. Maybe there are 2 flavors:

1. Add a `Manifold__Tooltip__Container` wrapper to use CSS positioning, or
2. Omit `Manifold__Tooltip__Container` to position manually (if you’re doing something special, such as adding a tooltip that needs to break outside of an `overflow: hidden` container)